src/server.c

```diff
@@ -2981,7 +2986,10 @@ sds genRedisInfoString(char *section) {
             "used_memory_rss_human:%s\r\n"
             "used_memory_peak:%zu\r\n"
             "used_memory_peak_human:%s\r\n"
+            "used_memory_peak_memkind:%zu\r\n"
```

Comment:
check - order of new specifiers ( lines 2989 2991 and 2992 ) with order of new variables ( lines 3017-3019 ). Line 2990 ?